### ISSUE TITLE
Guide how to work with server without using of built-in CrudManager

### ISSUE THREAD

**Issue by chuckn0rris** [bryntum/support#6571]
Many questions how to interact with server using own API and libs. What events listen to, how to set projectModel

**Comment by chuckn0rris** [comment]
one more request https://forum.bryntum.com/viewtopic.php?f=44&t=26984&p=134731#p134731

**Comment by MichalKolekMG** [comment]
Hi, is this guide still on the roadmap, please? I could use a deeper explanation of how to bypass CrudManager and wire to existing API.

**Comment by chuckn0rris** [comment]
@MichalKolekMG Yes it is still on roadmap! Are you using frameworks? We have guides for frameworks, I could point you on one of them in case.
